### PR TITLE
Implement Python Starter Snake

### PR DESCRIPTION
**Pull request by bevns**
This implements the Python Starter Snake to use as a starting point for the Summer League 2022 snake. 

closes #173

### CODE REVIEW

**Comment by wilsonwong1990** [comment]
Committed the installation of logic.

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,9 @@ WORKDIR /usr/src/app
 COPY Gemfile Gemfile.lock ./
 RUN bundle install --without dev
 
+RUN pip install flask
+RUN pip install logic
```
Comment (bevns) [inline Dockerfile:16]: @wilsonwong1990 can you help me understand why this is needed?

Comment (wilsonwong1990) [reply Dockerfile:16]: Yeah! When I setting up my own code spaces we can't use test.py without logic installed.

Comment (bevns) [reply Dockerfile:16]: 🤔 Were you getting an error of some kind? When I was looking at the tests here https://github.com/es-na-battlesnake/snakes/pull/173/files#diff-e784e09539c3775677aa46db6cdae7f99baf3487476267d7a9bb7ce351e99e39R11 I was under the impression that this was importing the `logic.py` file so we could reference the methods it contains. That logic would be different from what I think `pip install logic` would install.

**Comment by bevns** [comment]
Fixed the tests in https://github.com/es-na-battlesnake/snakes/pull/173/commits/0eccbfd5a8d8dca6ddcc89af0c84c09f1c11a1ef. 

```
# python3 src/tests.py -v
test_avoid_neck_all (__main__.AvoidNeckTest) ... ok
test_avoid_neck_down (__main__.AvoidNeckTest) ... ok
test_avoid_neck_left (__main__.AvoidNeckTest) ... ok
test_avoid_neck_right (__main__.AvoidNeckTest) ... ok
test_avoid_neck_up (__main__.AvoidNeckTest) ... ok

----------------------------------------------------------------------
Ran 5 tests in 0.000s

OK```

File: Dockerfile (hunk 1 above)
Comment (wilsonwong1990) [reply Dockerfile]: Working with @bevns looks like there was some error in my one codespace. We can remove `logic`

**Pull request merged**